Sources/Tracking/CustomerIO.swift

```diff
@@ -41,24 +41,13 @@ public extension CustomerIO {
         }
 
         let implementation = DataPipeline.initialize(moduleConfig: DataPipelineConfigOptions.Factory.create(sdkConfig: sdkConfig))
-
-        // Check if any unprocessed tasks are pending in the background queue.
         let newDiGraph = DIGraph(sdkConfig: sdkConfig)
-        let migrationAssistant = newDiGraph.dataPipelineMigrationAssistant
-        migrationAssistant.handleQueueBacklog()
-
-        initializeSharedInstance(with: implementation, diGraph: newDiGraph)
-
-        if sdkConfig.logLevel == .debug {
-            CustomerIO.shared.setDebugLogsEnabled(true)
-        }
-        if sdkConfig.autoTrackScreenViews {
-            // automatically add the AutoTrackingScreenViews plugin
-            DataPipeline.shared.analytics.add(plugin: AutoTrackingScreenViews(filterAutoScreenViewEvents: sdkConfig.filterAutoScreenViewEvents, autoScreenViewBody: sdkConfig.autoScreenViewBody))
-        }
+        initialize(implementation: implementation, diGraph: newDiGraph)
```

Comment:
I am guessing there is not change is the code but rather just made a method for the same changes?

Reply:
Yes, correct. No change in logic, only code moved to new `initialize` so it can be reused by methods initializing the module for testing.